application/src/main/java/org/togetherjava/tjbot/commands/code/CodeMessageAutoDetection.java

```diff
@@ -63,6 +66,18 @@ public void onMessageReceived(MessageReceivedEvent event) {
         codeMessageHandler.addAndHandleCodeMessage(originalMessage, true);
     }
 
+    private boolean isSendByTopHelper(MessageReceivedEvent event) {
+        if (Objects.requireNonNull(event.getMember()).getRoles().isEmpty()) {
+            return false;
+        }
+
+        return event.getMember()
+            .getRoles()
+            .stream()
+            .map(Role::getName)
+            .anyMatch(roleName -> roleName.matches("Top Helpers .+"));
```

Comment:
use config to take in the top helper role pattern

Reply:
config will now be used to get the pattern.